src/schema/base_schema.yaml

```diff
@@ -55,7 +55,7 @@ classes:
       xref:
         description: >-
           Holds one or more database cross references for an entity.
-        required: true
```

Comment:
Why was it changed to true?

Reply:
It was not changed to true. The true flag was deleted, because xref is generally optional.